### ISSUE TITLE
Run test with jest "Cannot find module 'ng2-charts'"

### ISSUE THREAD

**Issue by EPilz** [valor-software/ng2-charts#1352]
I cannot execute my jest tests with the newest version 3.0.1. I get following error:

```
    Cannot find module 'ng2-charts' from 'src/app/pages/text.component.ts'

    Require stack:
      src/app/pages/test-chart.component.ts
      src/app/pages/test.component.ts
      src/app/pages/test.spec.ts

    >  7 | import { BaseChartDirective } from "ng2-charts";
         | ^

   at Resolver.resolveModule (node_modules/jest-resolve/build/resolver.js:322:11)
      at Object.<anonymous> (src/app/pages/devices/device-detail/dialog-chart/dialog-chart.component.ts:7:1)
```

**Comment by btvanhooser** [comment]
getting the same problem on my end

**Comment by santam85** [comment]
Can you share a minimum reproduction project?

**Comment by btvanhooser** [comment]
I copied as much as I could configuration-wise from my site, and then used the demo (https://valor-software.com/ng2-charts/#LineChart) for the UI/logic to ensure it wasn't an issue with that.

Just running the tests via `npm run test` will show the issue as 
![image](https://user-images.githubusercontent.com/13162723/146594345-0b7ecdf0-f2f7-4f15-89c1-181988d6bfa2.png)


https://github.com/btvanhooser/ng2-charts-issue

**Comment by santam85** [comment]
What a headache! Did some experiments on your repo and I am thinking `jest-preset-angular` is still not supporting libraries with partial ivy compilation enabled. Trying to see if with custom configuration (enabling ESM) I am able to run it.

**Comment by btvanhooser** [comment]
sweet this helped out. I actually just added the 
`'ng2-charts': '<rootDir>/node_modules/ng2-charts/fesm2020/ng2-charts.mjs'`
line to the existing jest config in the repo and it worked like a charm. I ran into another error, but that was solved by updating this package since it was there error found in #1354 and that was resolved in v3.0.2. Then I got ANOTHER error, but a quick search revealed that it was caused by using the "ResizeObserver". To fix that, I just needed to add this to the setup-jest.ts file in my project to stub that out:
```JS
window.ResizeObserver =
    window.ResizeObserver ||
    jest.fn().mockImplementation(() => ({
      disconnect: jest.fn(),
      observe: jest.fn(),
      unobserve: jest.fn(),
    }));
```
